pkg/common/common.go

```diff
@@ -203,18 +203,35 @@ func parseMask(ipaddr string) (mask string, err error) {
 // GetIPListWithMaskFromString returns ip and mask in string form found in input string
 // A return value of nil indicates no match
 func GetIPListWithMaskFromString(input string) (string, error) {
-	re := regexp.MustCompile(`^([01]?\d\d?|2[0-4]\d|25[0-5])(?:\.(?:[01]?\d\d?|2[0-4]\d|25[0-5])){3}(?:/[0-2]\d|/3[0-2])?$`)
-	validated := re.FindAllString(input, 1)
-	if validated != nil {
-		mask, err := parseMask(validated[0])
+	// Split the IP address and subnet mask if present
+	parts := strings.Split(input, "/")
+	ip := parts[0]
+	result := net.ParseIP(parts[0])
+	if result == nil {
+		return "", errors.New("doesn't seem to be a valid IP")
+	}
+	if len(parts) > 1 {
```

Comment:
can this check be done before net.ParseIP(parts[0]) ??

Reply:
Based on the use case of this environment variable in the customer's production setup, it is more common for them to pass plain IP addresses without subnet masks, which increases the likelihood of entering invalid IP addresses wrt invalid mask ID or anything else. So I thought of putting as a second check.